Documentation/configuration.md

```diff
@@ -149,7 +149,110 @@ In `/etc/rkt/auth.d/specific-tectonic.json`:
 }
 ```
 
-The result is that when downloading data from `kubernetes.io` we still
-send `Authorization: Bearer common-token`, but when downloading from
-`coreos.com` - `Authorization: Basic Zm9vOmJhcg==`. And for
-`tectonic.com` - `Authorization: Bearer tectonic-token`.
+The result is that when downloading data from `kubernetes.io` `rkt`
+still sends `Authorization: Bearer common-token`, but when downloading
+from `coreos.com` - `Authorization: Basic Zm9vOmJhcg==` (`foo:bar`
+encoded in base64). And for `tectonic.com` - `Authorization: Bearer
+tectonic-token`.
+
+### rktKind: `dockerAuth`
+
+This kind of configuration is used to set up necessary credentials
+when downloading data from docker indices. The configuration files
+should be placed inside either `auth.d` or `docker.d` subdirectories
+(that is - in `/usr/lib/rkt/{auth.d,docker.d}` or in
+`/etc/rkt/{auth.d,docker.d}`).
+
+#### rktVersion: `v1`
+
+##### Description and examples
+
+This version of `dockerAuth` configuration specifies two additional
+fields: `indices` and `credentials`.
+
+The `indices` field is an array of strings describing docker indices
+for which following credentials should be used. A short list of
+popular docker indices is below. This field has to be specified and
+cannot be empty.
+
+`credentials` field holds the necessary data to authenticate against
+docker index. This field has to be specified and cannot be empty.
+
+Currently docker indices only support basic HTTP authentication, so
+`credentials` field has two subfields - `user` and `password`. These
+fields have to be specified and cannot be empty.
+
+Some popular docker indices:
+* index.docker.io (this is is used when no docker index is specified
+  in URL, like in `docker://redis`)
+* quay.io
+* gcr.io
+
+Example of dockerAuth config:
+```
+{
+	"rktKind": "dockerAuth",
+	"rktVersion": "v1",
+	"indices": ["index.docker.io", "quay.io"],
```

Comment:
(seems like docker/docker source code is mixed, clearest doc reference I can find is https://github.com/docker/docker/tree/master/docs/sources/terms)